storage/postgres/migrations/20250312074317_add_plugins.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS plugins (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    description TEXT,
```

Comment:
description TEXT NOT NULL

Reply:
https://github.com/LimeChain/vultiserver-plugin/pull/40/commits/dfb9c1ee6e0cb3ecf077f736e5f8aa57b1baed1c

Reply:
changed [here](https://github.com/LimeChain/vultiserver-plugin/pull/40/commits/2ca0ddeff28b097570b7f01ea6a793497b35b04a)